### DISCUSSION TITLE
Web wallet docker-compose folder missing ?

### DISCUSSION THREAD

**Discussion by PekkaKW** [walt-id/.github#75] [Help] [answered]
Hi, I downloaded waltid-identity last Thursday and have been setting it up on my local server. I had some issues with docker-compose (issuer config was pointing to waltid-issuer instead of waltid-issuer-api). And then was facing some issues with - 'pushPublicKey': Missing from config. I left this issue and did some skiing instead :) 
Now I am back trying to get this wallet up..  But now, when I look at this in github, I see that docker-compose is totally missing in waltid-identity/waltid-web-wallet/ ? Is that intentional ? If so then please update also https://docs.walt.id/v/apps/solutions/web-wallet/quick-start, which still refers to that folder.

**Comment by PekkaKW** [comment]
Okk. I can see that now readme is updated with [feat: Added wallet-backend startup command for docker].

**Comment by taminobaumann** [comment, marked as answer]
Hey @PekkaKW, 
Were you able to get the wallet up and running using the docker-compose command below, executed from the root of the waltid-identity?


```
cd docker-compose && docker-compose up
```
